packages/shared/friends/sagas.ts

```diff
@@ -176,7 +176,10 @@ function* initializeFriendsSaga() {
     const isGuest = yield select(getIsGuestLogin)
 
     // guests must not use the friends & private messaging features
-    if (isGuest) return
+    if (isGuest) {
+      getUnityInstance().InitializeChat({ totalUnseenMessages: 0 })
```

Comment:
It might be better for present usage as well as the future to have some kind of flag or kind here, i.e. `isGuest: true | false` or `kind: `User.WALLET | User.Guest`, to have a better understanding of how to handle this on the renderer side.

Reply:
it is useful, but not needed at the moment though